pkg/manifest/tests/results/aliases.yaml

```diff
@@ -8,7 +8,7 @@
     - "/root2/rebase/README/"
     - "/root3/rebase_slash/README/"
 - file: _index.md
-  processor: markdown
+  processor: downloader
   type: file
   source: https://github.com/gardener/docforge/blob/master/contents/website/blog/2024/_index.md
```

Comment:
Can you make all `md` files under `pkg/manigest/tests` to `.txt` and replace `_inted.md` to `README.txt`?